src/icon-button.css.ts

```diff
@@ -1,7 +1,511 @@
+            [backgroundActive]: skinVars.colors.backgroundContainerPressed,
+            [iconColor]: skinVars.colors.error,
+        },
+    },
+    'danger-soft-inverse': {
+        vars: {
+            [background]: skinVars.colors.buttonLinkDangerBackgroundInverse,
+            [backgroundHover]: skinVars.colors.backgroundContainerHover,
+            [backgroundActive]: skinVars.colors.backgroundContainerPressed,
+            [iconColor]: skinVars.colors.error,
+        },
+    },
+    'danger-soft-alternative': {
+        vars: {
+            [background]: skinVars.colors.errorLow,
+            [backgroundHover]: skinVars.colors.backgroundContainerHover,
+            [backgroundActive]: skinVars.colors.backgroundContainerPressed,
+            [iconColor]: skinVars.colors.error,
+        },
+    },
+
+    'danger-default-default': {
+        vars: {
+            [background]: 'transparent',
+            [backgroundHover]: skinVars.colors.backgroundContainerHover,
+            [backgroundActive]: skinVars.colors.backgroundContainerPressed,
+            [iconColor]: skinVars.colors.error,
+        },
+    },
+    'danger-default-inverse': {
+        vars: {
+            [background]: skinVars.colors.buttonLinkDangerBackgroundInverse,
+            [backgroundHover]: skinVars.colors.backgroundContainerHover,
+            [backgroundActive]: skinVars.colors.backgroundContainerPressed,
+            [iconColor]: skinVars.colors.error,
+        },
+    },
+    'danger-default-alternative': {
+        vars: {
+            [background]: 'transparent',
+            [backgroundHover]: skinVars.colors.backgroundContainerHover,
+            [backgroundActive]: skinVars.colors.backgroundContainerPressed,
+            [iconColor]: skinVars.colors.error,
+        },
+    },
+
+    'brand-media': {
+        vars: {
+            [background]: applyAlpha(skinVars.rawColors.inverse, 0.7),
+            [backgroundHover]: applyAlpha(skinVars.rawColors.inverse, 0.9),
+            [backgroundActive]: applyAlpha(skinVars.rawColors.inverse, 1.0),
+            [iconColor]: skinVars.colors.brand,
+        },
+    },
+    'neutral-media': {
+        vars: {
+            [background]: applyAlpha(skinVars.rawColors.inverse, 0.7),
+            [backgroundHover]: applyAlpha(skinVars.rawColors.inverse, 0.9),
+            [backgroundActive]: applyAlpha(skinVars.rawColors.inverse, 1.0),
+            [iconColor]: 'black',
+        },
+    },
+    'danger-media': {
+        vars: {
+            [background]: applyAlpha(skinVars.rawColors.inverse, 0.7),
+            [backgroundHover]: applyAlpha(skinVars.rawColors.inverse, 0.9),
+            [backgroundActive]: applyAlpha(skinVars.rawColors.inverse, 1.0),
+            [iconColor]: skinVars.colors.error,
+        },
+    },
+});
+
+export const disabled = style({
+    opacity: 0.5,
+});
+
+export const isLoading = style({});
+export const overlayContainer = style({});
+
+const transitionTiming = '0.1s ease-in-out';
```

Comment:
I think this is too fast, the transition is barely noticeable. Consider something like 0.3s

Reply:
this was an old bug that we never noticed until now. Good catch! It should be fixed now